src/run.php

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+// src/run.php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+
+$setAutoloader();
+$terminus = new Terminus();
+$command_factory = new AnnotatedCommandFactory();
```

Comment:
Using annotated command directly rather than using the Robo runner is fine. Since you have your own Symfony Application class, though, maybe you should move all of this init code into some method in the Terminus class, and call your initializer from `$terminus->run()`.